src/sql/workbench/contrib/query/browser/query.contribution.ts

```diff
@@ -139,6 +139,16 @@ actionRegistry.registerWorkbenchAction(
 	RunCurrentQueryWithActualPlanKeyboardAction.LABEL
 );
 
+actionRegistry.registerWorkbenchAction(
+	SyncActionDescriptor.create(
+		CopyQueryWithResultsKeyboardAction,
+		CopyQueryWithResultsKeyboardAction.ID,
+		CopyQueryWithResultsKeyboardAction.LABEL,
+		{ primary: KeyMod.CtrlCmd | KeyCode.KEY_M }
```

Comment:
This key binding is already used for RunCurrentQueryWithActualPlan https://github.com/microsoft/azuredatastudio/blob/main/src%2Fsql%2Fworkbench%2Fcontrib%2Fquery%2Fbrowser%2Fquery.contribution.ts#L132

Reply:
please work with PM to define the shortcut key @gupta1

Reply:
Sure, changing it to Ctrl+Alt+C for now since Ctrl+Shift+C  which is used in Kusto Explorer is already used for something else in ADS. I can change it later if needed once @MsSQLGirl and Raden are back from vacation next week.